### DISCUSSION TITLE
Search saved comments or posts

### DISCUSSION THREAD

**Discussion by voxvargr** [Docile-Alligator/Infinity-For-Reddit#874] [Ideas]
It should be loved by all

**Comment by Wladefant** [comment]
The duplicate Reddit link is here: https://www.reddit.com/r/Infinity_For_Reddit/comments/vxb05c/search_through_saved_posts_and_comments/

**Comment by hugo9655** [comment]
bump. id absolutely love to see this implemented. searching on reddits website is truly awful. this feature would be so helpful for referencing and finding older posts.

**Comment by edgan** [comment]
`Continuum` now has Saved search.

**Comment by hugo9655** [comment]
OP please keep this open for Infinity users.
